base/c.jl

```diff
@@ -23,7 +23,7 @@ if is_windows()
 else
     typealias Clong Int
     typealias Culong UInt
-    typealias Cwchar_t Int32
+    typealias Cwchar_t UInt32
```

Comment:
Consistency with windows is not the point for this definition. This is what the C type is.